src/utils.ts

```diff
@@ -0,0 +1,564 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  Block,
+  ApiResponse,
+  FullBlock,
+  Input,
+  Output,
+  DecodedScript,
+  FullTx,
+  Token,
+  StatusEvent,
+  PreparedTx,
+  PreparedInput,
+  PreparedOutput,
+} from './types';
+import AWS from 'aws-sdk';
+import axios from 'axios';
+import dotenv from 'dotenv';
+import { constants } from '@hathor/wallet-lib';
+
+dotenv.config();
+
+AWS.config.update({
```

Comment:
I feel we should move all API methods to an `api.js` file. We can split it into `api/lambda.js` and `api/fullnode.js` if it makes sense.

Reply:
I agree.

Done in c5e1e4b4b2b7be18707e3e1c61f18e5bd758fe49

Reply:
I really liked it. Thanks!